shared/overview/2.x.md

```diff
@@ -4,52 +4,55 @@
-buffer in order to avoid wear on the flash storage used by most of the supported boards. However, persistent logging can be enabled by setting
-the `"persistentLogging": true` key in the `config.json` file in the boot partition of the device. The logs can be accessed [via the host OS][ssh-host] at `/var/log/journal`.
+Production images disable passwordless root access, and an SSH key must be [added][config-json-ssh] to `config.json` to access a production image using a direct SSH connection. You may still access a production image by tunneling SSH through the {{ $names.company.lower }} VPN via the CLI (using `balena ssh <uuid>`) or the {{ $names.cloud.lower }} [web terminal][ssh-host]. To use SSH via the VPN, you need to have an SSH key configured on your development machine and [added][ssh-key-add] to the {{ $names.cloud.lower }} dashboard.
+
+In both development and production versions of {{ $names.os.lower }}, logs are written to an 8 MB journald RAM buffer in order to avoid wear on the flash storage used by most of the supported boards.
+
+To persist logs on the device, enable persistent logging via the [fleet configuration variable][fleet-configuration] in the {{ $names.cloud.lower }} dashboard, or prior to device provisioning setting the `"persistentLogging": true` [key][config-json-logging] in `config.json`. The logs can be accessed via the host OS at `/var/log/journal`. For versions of {{ $names.os.lower }} < 2.45.0, persistent logs are limited to 8 MB and stored in the state partition of the device. {{ $names.os.upper }} versions >= 2.45.0 store a maximum of 32 MB of persistent logs in the data partition of the device.
+
+Both development and production versions of {{ $names.os.lower }} allow the setting of a custom [hostname][config-json-hostname] via `config.json`, by setting `"hostname": "my-new-hostname"`. Your device will then broadcast (via Avahi) on the network as `my-new-hostname.local`. If you don't set a custom hostname, the device will default to `<short-UUID>.local`. You can also set a custom hostname via the [Supervisor API][supervisor-api] on device.
 
-Both Prod and Dev variants will also allow the setting of a custom hostname via the `config.json`, just add `"hostname": "my-new-hostname"`. Your device will then broadcast (via Avahi) on the network as `my-new-hostname.local`. If you don't set a custom hostname, the device will default to `<short-UUID>.local`.
+On production images, nothing is written to tty1, on boot up you should only see the {{ $names.company.lower }} logo on the HDMI screen, and this will persist until your application code takes over the framebuffer. If you would like to replace the {{ $names.company.lower }} logo with your own custom splash logo, then you will need to replace splash/resin-logo.png file that you will find in the [first partition][parition] of the image (boot partition or `resin-boot`) with your own logo.
```

Comment:
"replace splash/resin-logo.png file" -> "replace the splash/resin-logo.png file" ?